extension/contentfragment/content/src/content/jcr_root/apps/core/wcm/extension/components/contentfragment/v1/contentfragment/_cq_dialog/.content.xml

```diff
@@ -4,7 +4,8 @@
     jcr:title="Content Fragment"
     sling:resourceType="cq/gui/components/authoring/dialog"
     extraClientlibs="[core.wcm.extension.components.contentfragment.v1.dialog]"
-    helpPath="https://www.adobe.com/go/aem_cmp_contentfragment_v1">
+    helpPath="https://www.adobe.com/go/aem_cmp_contentfragment_v1"
+    trackingFeature="core-component:extension:contentfragment:v1">
```

Comment:
As we want to remove the extension concept, maybe it would be best not to represent it here, otherwise we would split the data set when we come to remove it?